README.md

```diff
@@ -133,8 +133,8 @@ This assumes that the pre-production server is otherwise ready.
   -``sudo -u apache /usr/local/bin/wp config set WP_DEBUG false --raw --path=/var/www/html``
   - ``sudo -u apache /usr/local/bin/wp super-cache flush --path=/var/www/html``
  - Make sure the menu background is now black.
- - login to the proxy: ("kielipankki-proxy-prod": ssh cloud-user@195.148.30.210)
- - change the proxy settings in the proxy's Ansible configuration under group-vars/all to point the variable `portal_server_ip` to point to the pre-production server's INTERNAL IP (7/2020 that is: 192.168.1.8). The new server is immediately in use and now considered production.
+ - update IP settings in `[Kielipankki-proxy](https://github.com/CSCfi/Kielipankki-proxy)` (see README.md there)
```

Comment:
```suggestion
 - update IP settings in [Kielipankki-proxy](https://github.com/CSCfi/Kielipankki-proxy) (see README.md there)
```
Cant have the \`'s here or we will get literally `[Kielipankki-proxy](https://...`